voltdb/assets/configuration/spec.yaml

```diff
@@ -26,6 +26,46 @@ files:
               type: boolean
               example: false
 
+          - name: statistics_components
+            description: |
+              See: https://docs.voltdb.com/UsingVoltDB/sysprocstatistics.php
+              The VoltDB integration does not collect metrics from all components by default, some are redundant, and
+              some can have a very high cardinality associated.
```

Comment:
You are explaining what it does not do by default but not what this list means, even if it's obvious.